### PR TITLE
Fixing WriteAllLines issue

### PR DESCRIPTION
**Pull request by KingLuigi4932**
Since IL2CPU doesn't support arrays as `IEnumerable<T>`
I changed WriteAllLines code
and convert string[] into a string ends with '\r\n' based on windows for creating New Line

### CODE REVIEW

File: source/Cosmos.System2_Plugs/System/IO/FileImpl.cs [hunk 1]
```diff
@@ -119,19 +119,14 @@ public static string[] ReadAllLines(string aFile)
 
             return result;
         }
-
-        public static void WriteAllLines(string aFile, IEnumerable<string> contents)
+
```
Comment (jp2masa) [inline source/Cosmos.System2_Plugs/System/IO/FileImpl.cs]: Nit: trim whitespace.

File: source/Cosmos.System2_Plugs/System/IO/FileImpl.cs [hunk 2]
```diff
@@ -119,19 +119,14 @@ public static string[] ReadAllLines(string aFile)
...
-            {
-                text = string.Concat(text, line, Environment.NewLine);
-            }
-
+            string text = String.Join(Environment.NewLine, contents);
+            
             Global.mFileSystemDebugger.SendInternal("Writing contents");
             Global.mFileSystemDebugger.SendInternal(text);
-
+
```
Comment (jp2masa) [inline source/Cosmos.System2_Plugs/System/IO/FileImpl.cs]: Nit: trim whitespace.

Comment (KingLuigi4932) [reply source/Cosmos.System2_Plugs/System/IO/FileImpl.cs]: Working when trying to use the codes in the kernel :)

Comment (KingLuigi4932) [reply source/Cosmos.System2_Plugs/System/IO/FileImpl.cs]: Means it works with Writing Test and Reading Test

Comment (fanoI) [reply source/Cosmos.System2_Plugs/System/IO/FileImpl.cs]: You need to add a test in the FatTestRunner so AppVoyer will compile it in any PR and we known automatically that it did not broke.

TestRunner is easier for you too as now you need to have an external kernel, compile Cosmos anytime, Compile the Kernel... right?

Comment (KingLuigi4932) [reply source/Cosmos.System2_Plugs/System/IO/FileImpl.cs]: Yes, You mean I Need to Create A Tester in this pull request

**Pull request merged**